webrender/src/frame_builder.rs

```diff
@@ -1155,10 +1155,17 @@ impl FrameBuilder {
             }
         }
 
+        let color = match font.render_mode {
+            FontRenderMode::Bitmap => ColorF::new(1.0, 1.0, 1.0, color.a),
+            FontRenderMode::Subpixel |
+            FontRenderMode::Alpha |
+            FontRenderMode::Mono => *color,
+        };
```

Comment:
As a minor style point, this could just be a ternary:

`let color = if font.render_mode == FontRenderMode::Bitmap { ... } else { ... };`

I don't expect this should need to be updated for other font modes.